packages/component_code_gen/README.md

```diff
@@ -0,0 +1,36 @@
+# AI Code Gen for Pipedream Components
+
+Generate components using OpenAI GPT-4.
+
+
+### Run
+
+```
+SCRIPT=generate_action.py
+APP=slack
+PROMPT="how to send myself a direct message?"
+poetry run python3 "$SCRIPT" --app "$APP" "$PROMPT"
+poetry run python3 "$SCRIPT" --app "$APP" "$PROMPT" --verbose # print debug logs
+```
+
+
+### Installation
+
+1. Install poetry: follow instructions at https://python-poetry.org/docs/#installation
+
+2. Run install:
+```
+poetry install
+```
+
+
+### Setup
+
+Create a `.env` file
+
+Add these API Keys:
```

Comment:
One thing we do in other repos is provide a `.env.example` file and tell people to copy that to a `.env` file and fill in the values. Just slightly easier than having the dev copy + paste these one by one